version.php

```diff
@@ -25,9 +25,9 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version = 2024050801;
+$plugin->version = 2024050802;
```

Comment:
Why a version bump?  There is nothing that actually needs it here.  Also version bumping in a code change commit makes it more difficult to back port.  Better to version bump a new development sequence, commit, then add code required in separate commits.  Then even if code needs the version to be bumped then when the client installs the tagged release, this happens.  Thus leaving only devs to cope, which they can by down then upgrading.